packages/webservice/app/src/test/kotlin/com/yahoo/navi/ws/test/unit/SameUserTest.kt

```diff
@@ -27,7 +26,7 @@ class SameUserTest {
         val matchingUserId = "test user"
 
         // Mock a request user
-        val requestUser = Mockito.mock(MappedLoginService.UserPrincipal::class.java)
+        val requestUser = Mockito.mock(Principal::class.java)
```

Comment:
Can this be something like: 
```kt
Mockito.`when`(requestScope.getUser().getOpaqueUser()).thenReturn(() -> matchingUserId)
```

Reply:
That doesn't seem to satisfy the contract of that returning a principal

Reply:
Sorry that was the java version. I beleive the kotlin version is:

```
Mockito.`when`(requestScope.getUser().getOpaqueUser()).thenReturn(Principal { matchingUserId })
```